inst/doc/custom_data_sources.Rmd

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-package-vignettes

So...my understanding is that CRAN (and also GRAN) does not rebuild vignettes, so if *you* don't put all the relevant files in inst/doc, they won't show up in the package index help file:

![image](https://user-images.githubusercontent.com/1105215/29465616-8393818c-83ff-11e7-8be6-0504db9b5d00.png)

However, we keep them in the vignette folder because the `check` will test all the code in the vignettes....

I might be wrong on this...I'll try to double check.